### ISSUE TITLE
Container Logging?

### ISSUE THREAD

**Issue by orchestrate8D** [a2o/snoopy#111]
## Checklist before I submit this issue report

I confirm that:
- [ x] I have tested this with the latest version of Snoopy available @ https://github.com/a2o/snoopy#i1-version-information-and-download-links (not the latest Snoopy version packaged by your favorite distribution, it might be out of date)
- [x ] I have read Snoopy documentation @ https://github.com/a2o/snoopy/blob/master/README.md & https://github.com/a2o/snoopy/tree/master/doc
- [ x] I have checked FAQ entries @ https://github.com/a2o/snoopy/blob/master/doc/FAQ.md
- [ x] I have checked existing Snoopy issues for duplicates and found none @ https://github.com/a2o/snoopy/issues?q=is%3Aissue

## Issue description - Environment basics

(replace 'FILLIN' with actual values from your environment)

| Key                                 | Value             | Comments                         |
| ----------------------------------- | ----------------- | ---------------------------------|
| Architecture:                       | x86_64            | x86 is not supported anymore     |
| Linux distribution:                 | Debian Jessie/Proxmox 3.4        | |
| Distribution version:               | 8 / 3.4           | |
| Snoopy version:                     | Latest Stable           | |
| Snoopy config file was used:        | Yes   | snoopy-install.sh enables this by default |
| Snoopy threading support enabled:   | no  | not enabled by default as of v2.4.4       |

## Issue description - How was Snoopy installed

FILLIN
(describe installation method - snoopy-install.sh, custom built (provide configure arguments), distro package)

Used recommended snoopy-install.sh

## Issue description - Describe the problem you are having
Proxmox is an openvz management tool. USing auditd I was able to see syscalls from containers in the host but it lacked the only_tty filter. With snoopy I don't see anything coming from within the containers. Any reason why?

## Expected result

Logging of commands in containers

## Actual result
Nothing

## Steps to reproduce
Install proxmox (really easy from ISO)
Install snoopy
create a basic openvz container.

**Comment by bostjan** [comment]
I am not intimately familiar with Proxmox, but if it is anything similar to LXC, it cleans the environment before starting processes in a "containerized" environment (chroot, mount, proc, dev namespace).

Therefore, Snoopy's LD_SO_PRELOAD environment variable does not leak into the container from the host, nor is container host's /etc/ld.so.preload file visible from within container environment.

This is a simple no-can-do for Snoopy. You would need to use kernel facilities to trigger command logging, and dig into namespaces to gather relevant information for other data collectors (uid->username  conversion etc).

In short - won't do.

I am not trying to discourage you from implementing this feature yourself. I can see the value of logging all the stuff that goes on in containers, and might even use the solution myself if need will arise eventually. I currently see no real use for it.

(I guess that $10-50k investment into time of some experienced C developer could get you the solution you are looking for.)
